### PR TITLE
Cleanup for V1 Release

### PR DESCRIPTION
**Pull request by alexgb**
Readme cleanup in preparation for 1.0 release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,29 +91,20 @@ to React without the hard requirement to start with leaf components. It is
...
-community has still developed some conventions over time.
-
-For React component files, the widely adopted convention is PascalCase,
-including
-[Airbnb](https://github.com/airbnb/javascript/tree/master/react#naming). So we
-have added support for this convention.
-
-In short, you can name your JSX files in `PascalCase`, in addition to
-`snake-case`.
+You can name your react component files using either the ember convention of
```
Comment (pswai) [inline README.md]: I think it's good to be consistent and capitalise `React` and `Ember`

File: README.md [hunk 2]
```diff
@@ -178,41 +152,6 @@ this repo.
 Completed {{completedTodos.length}} todos
 ```
 
-#### app/routes/application.js
```
Comment (pswai) [inline README.md]: Removing the route and controller makes it so much cleaner (y)

**Comment by alexgb** [comment]
@pswai added changelog. It's format is inspired by https://keepachangelog.com/en/1.0.0/, but I've omitted the changed/removed/added blocks.

**Comment by pswai** [comment]
@alexgb I think that's pretty good :)

And i just recalled that we need a migration path for the breaking change, here's a draft I can think of:

## Migrating to 1.0

[`ember-browserify`](https://github.com/ef4/ember-browserify) has been migrated to [`ember-auto-import`](https://github.com/ef4/ember-auto-import). To migrate to 1.0, there are several steps you need to do:

1. Remove `ember-browserify` from `package.json` (if no other addon is using)
2. Install latest `ember-cli-react` with blueprint triggered
3. Remove `npm:` prefix from all import statements

Then you are good to go :)

**Comment by alexgb** [comment]
@pswai 1.0 is released! Thanks for all contributions.

**Comment by pswai** [comment]
Awesome, thanks!

**Pull request merged**